pipeline/inputs/aggregator_test.go

```diff
@@ -343,7 +343,7 @@ func TestAggregator_AddReport(t *testing.T) {
 		if err := a.AddReport(metrics.MetricReport{
 			Name:      "int-metric",
 			StartTime: time.Unix(0, 0),
-			EndTime:   time.Unix(1, 0),
+			EndTime:   time.Unix(2, 0),
```

Comment:
Can you add an additional report to test the expanding of start time as well? This is currently only testing the end time.